Deeploy/Targets/Generic/Templates/FloatPowTemplate.py

```
@@ -0,0 +1,48 @@
+
+from Deeploy.DeeployTypes import NetworkContext, NodeTemplate, OperatorRepresentation
+
+
+class _PowTemplate(NodeTemplate):
+
+    def alignToContext(self, ctxt: NetworkContext,
+                       operatorRepresentation: OperatorRepresentation) -> Tuple[NetworkContext, Dict, List[str]]:
+
+        # Get input and output tensors
+        data_in = ctxt.lookup(operatorRepresentation['data_in'])
+        data_out = ctxt.lookup(operatorRepresentation['data_out'])
```

Ruff (0.14.5)

Local variable `data_out` is assigned to but never used

Remove assignment to unused variable `data_out`

(F841)